cloud-http-get-test.yaml

```diff
@@ -0,0 +1,12 @@
+meta:
```

Comment:
Looks like this file is not used anywhere

Reply:
Yeah I was using this to test locally! Thanks!